spec/cpu.go

```diff
@@ -0,0 +1,82 @@
+package spec
+
+import (
+	"bufio"
+	"context"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/mackerelio/golib/logging"
+	mackerel "github.com/mackerelio/mackerel-client-go"
+)
+
+// CPUGenerator collects CPU specs
+type CPUGenerator struct {
+}
+
+// TODO: タグこれでいいのかな
+var cpuLogger = logging.GetLogger("spec.cpu")
```

Comment:
LGTM

Reply:
Thanks, I'm going to remove this comment.